.github/workflows/prepare-release.yml

```diff
@@ -6,63 +6,97 @@ on:
     # the minor version number and set the patch number to 0.
     inputs:
       candidate-stable:
-        description: Release candidate version (stable, like 1.3.0). Don't include a leading `v`.
-
-      current-stable:
-        required: true
-        description: Current version (stable, like 1.2.0). Don't include a leading `v`.
+        description: Release candidate version bump (stable).
+        type: choice
+        options:
+          - minor
+          - patch
+          - "no bump"
+        default: minor
 
       candidate-beta:
-        description: Release candidate version (beta, like 0.96.0). Don't include `v`.
-
-      current-beta:
-        required: true
-        description: Current version (beta, like 0.95.1). Don't include `v`.
+        description: Release candidate version bump (beta).
+        type: choice
+        options:
+          - minor
+          - patch
+          - "no bump"
```

Comment:
not sure if this is needed or even possible for the beta component set

Reply:
Maybe in the far future, but for now this is not something that would happen ever

Reply:
ok then i'll remove the option
```suggestion
```